scripts/apps/web-publisher/styles/_helpers_variables.sass

```diff
@@ -0,0 +1,45 @@
+$superdesk-green:       rgba(49,172,102,1);
```

Comment:
Superdesk already has its own styling, talk to @fritzSF or @pavlovicnemanja if you need help with branding, I think adding any theming at module level will create problems later.

You just need to `@import '~variables.scss'` - why is that not working?